client/lib/performance-tracking/index.ts

```diff
@@ -6,5 +6,7 @@ export { withStopPerformanceTrackingProp } from './with-stop-performance-trackin
 export { default as PerformanceTrackerStop } from './performance-tracker-stop';
 
 export type PerformanceTrackProps = {
-	stopPerformanceTracking: () => void;
+	stopPerformanceTracking: (
+		metadata: { [ key: string ]: string | boolean | number } = {}
+	) => void;
```

Comment:
Got a Typescript error compiling this.

The fact that it defaults to `{}` isn't part of the type signature, but the fact it's optional is. The way to express that is to use the `?` operator.

```suggestion
	stopPerformanceTracking: ( metadata?: { [ key: string ]: string | boolean | number } ) => void;
```

Reply:
Already fixed by the time I submitted the comment :)

Reply:
I fixed it a few minutes ago but forget to push (🤦 ). Your comment made me remember it :)